docs/content/quickstart/installation.md

```diff
@@ -87,72 +87,72 @@ k3s:
+The CLI allows to register a node with a screenshot, an image, or a token. During pairing, the configuration is sent over, and the node will continue the installation process.
 
-In a terminal in your desktop/workstation, run:
+In a terminal window from your desktop/workstation, run:
 
 ```
 kairos register --reboot --device /dev/sda --config config.yaml
 ```
 
-Note:
-- By default the CLI will take automatically a screenshot to get the QR code. Make sure to get that fit into the screen. Alternatively an image path or a token can be supplied via args (e.g. `kairos register /img/path` or `kairos register <token>`)
-- The `--reboot` flag will make the node reboot automatically after the installation is completed
-- The `--device` flag will instruct to install kairos in the specified drive, replace `/dev/sda` with your drive. It will take over and overwrite any content, please be cautious.
-- The `--config` flag is used to specify the config file to drive the installation
+**Note**:
+- By default, the CLI will take automatically a screenshot to get the QR code. Make sure to get that fit into the screen. Alternatively, an image path or a token can be supplied via arguments (e.g. `kairos register /img/path` or `kairos register <token>`).
+- The `--reboot` flag will make the node reboot automatically after the installation is completed.
+- The `--device` flag will instruct to install Kairos in the specified drive, replace `/dev/sda` with your drive. It will take over and overwrite any content, please be cautious.
+- The `--config` flag is used to specify the config file to drive the installation.
 
 Wait now for few minutes for the config to propagate to the node, the installation should start then and automatically reboot afterward.
 
-### Accessing the node
+### Accessing the Node
 
-After boot the node will start and load into the system. When the console is available we should be already able to SSH.
+After a boot, the node will start and load into the system. When the console is available, we should be already able to SSH.
 
-To access to the host, login as `kairos`:
+To access to the host, log in as `kairos`:
 
 ```bash
 ssh kairos@IP
 ```
 
-Note:
-- `sudo` is configured for the `kairos` user
+**Note**:
+- `sudo` is configured for the Kairos user.
 
-You should be greeted with a welcome message:
+You will be greeted with a welcome message:
 
 ```
-Welcome to kairos!
+Welcome to Kairos!
 
 Refer to https://kairos.io for documentation.
-kairos@kairos:~> 
+kairos@kairos:~>
 ```
 
-At this point, it can take few moments to get the k3s server running, but eventually we should be able to inspect the service and see k3s running, for example with systemd-based flavors:
+At this point, it can take few moments to get the K3s server running, but eventually we should be able to inspect the service and see K3s running. For example with systemd-based flavors:
```

Comment:
```suggestion
At this point, it can take a few moments to get the K3s server running. However, you should be able to inspect the service and see K3s running. For example, with systemd-based flavors:
```